conda_config.py

```diff
@@ -0,0 +1,38 @@
+import yaml, argparse, os
+
+
+class CondaDumper(yaml.Dumper):
+
+    def increase_indent(self, flow=False, indentless=False):
+        return super(CondaDumper, self).increase_indent(flow, False)
+
+
+def write_conda_recipe(version):
+    """
+    Writes the EvalML recipe to build a conda package based on the version
+    Args:
+        version: The version of EvalML we are building with this feedstock
+
+    Returns:
+        None: Side effect of overwriting the existing meta.yaml in the feedstock
+    """
+
+    recipe_file_path = os.path.join(os.path.join(os.path.join(os.getcwd(), 'evalml-core-feedstock'), 'recipe'), 'meta'
+                                                                                                                '.yaml')
+    with open(recipe_file_path, 'rb') as config_file:
+        # Toss out the first line that declares the version since its not supported YAML syntax
+        next(config_file)
+        config = yaml.safe_load(config_file)
+        recipe_path = f'../feedstock_root/evalml'
+        config['source'] = {'path': recipe_path}
+        config['package']['version'] = version
+
+    with open(recipe_file_path, 'w') as recipe:
+        yaml.dump(config, recipe, default_flow_style=False, sort_keys=False, Dumper=CondaDumper)
+
+
+if __name__ == "__main__":
+    parser = argparse.ArgumentParser(description="Configure conda for local build. Run from the feedstock root")
+    parser.add_argument('version', help='The version of EvalML being built')
+    args = parser.parse_args()
+    write_conda_recipe(args.version)
```

Comment:
@ryan-s this is super cool!!

Please move this file to a different location instead of the top-level directory. Should it go in `.circleci/`? That would be my first inclination.